src/gui/FileBrowser.cpp

```diff
@@ -403,11 +340,11 @@ void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent * e )
 						this,
 					SLOT( sendToActiveInstrumentTrack() ) );
 		contextMenu.addAction( tr( "Open in new instrument-track/"
-								"Song-Editor" ),
+								"Song Editor" ),
 						this,
 					SLOT( openInNewInstrumentTrackSE() ) );
 		contextMenu.addAction( tr( "Open in new instrument-track/"
-								"B+B Editor" ),
+								"Pattern Editor" ),
```

Comment:
> Was the change in the view menu a rogue edit? I can submit rollback patches for that plus the change I made, if desired.

I'm not sure, but the vote seems to be to move ahead with "Pattern Editor" long term, so a separate fixup commit would be just fine until that happens (and its probably best to leave the name change to occur when the class rename occurs too).